web-ng/src/app/components/layer-dialog/layer-dialog.component.html

```diff
@@ -59,20 +67,20 @@
   </mat-dialog-content>
   <mat-dialog-actions fxLayout fxLayoutAlign="space-between">
     <div>
-      <button fxFlex="none" mat-icon-button type="button" (click)="onDeleteLayer()">
+      <button
+        fxFlex="none"
+        mat-icon-button
+        type="button"
+        (click)="onDeleteLayer()"
+      >
         <mat-icon>delete_forever</mat-icon>
       </button>
     </div>
     <div fxLayoutAlign="end">
-      <button
-        mat-raised-button
-        (click)="onClose()"
-        type="button"
-        class="btn-cancel"
-      >
+      <button mat-raised-button (click)="onClose()" class="btn-cancel">
         Cancel
       </button>
-      <button mat-raised-button color="primary" type="submit" class="btn-save">
+      <button mat-raised-button type="submit" class="btn-save">
```

Comment:
Please use the `color` attribute to set colors here and throughout instead of customizing CSS directly: https://material.angular.io/components/button/overview